sanic/config.py

```diff
@@ -45,8 +55,23 @@
     "WEBSOCKET_PING_TIMEOUT": 20,
 }
 
+# These values will be removed from the Config object in v22.6 and moved
+# to the application state
+DEPRECATED_CONFIG = ("SERVER_RUNNING", "RELOADER_PROCESS", "RELOADED_FILES")
+
+
+class CastRegistry(deque):
+    def add(self, cast: Callable[[str], Any]) -> None:
+        if cast in self:
+            raise SanicException(
+                f"Type {cast.__name__} has already been registered"
+            )
```

Comment:
Is throwing an exception when a you want to add a type converter which already in here wanted behaviour? 

When added duplicated type converters it would require some way check on which convertor its throwing the exception, after the dev figured that out it would need to try it again without that converter and if it goes wrong again it needs to remove that one again. 

What if it logged a warning with that someone tried to add a duplicate to the convertors instead of throwing an exception?

Reply:
Yeah, that makes sense. A warning is probably sufficient here.